app/models/trip.rb

```diff
@@ -18,9 +18,9 @@ class Trip < ApplicationRecord
   validates_inclusion_of :comfort, in: CAR_RATINGS
   validates_inclusion_of :state, in: STATES
   validates_inclusion_of :departure_date, in: Time.zone.today..Time.zone.today+1.year, message: "Mettre une date situé entre aujourd hui et dans 1 an."
-  validates_numericality_of :seats, { greater_than_or_equal_to: 1 }
-  validates_numericality_of :price, { greater_than_or_equal_to: 0 }
-  validates_numericality_of :age, allow_blank: true
+  validates_numericality_of :seats, only_integer: true, greater_than_or_equal_to: 1
+  validates_numericality_of :price, greater_than_or_equal_to: 0
+  validates_numericality_of :age, only_integer: true, allow_blank: true, greater_than: 0, less_than: 100
```

Comment:
Only euros no cents ?

Reply:
its the age so not a float

Reply:
i fixed it :)

Reply:
Could we add the "0" for the nb of seats?
cf : https://github.com/covoiturage-libre/covoiturage-libre/issues/141

Reply:
Good proposition, but I won't do it here. I prefer to keep PRs synthetic, this is out of the scope.
Maybe in another one ?

Reply:
@webplate Pourquoi ne pas mettre un âge supérieur à 18 par ailleurs?